recipes/mira/meta.yaml

```diff
@@ -1,11 +1,12 @@
 about:
   home: https://sourceforge.net/p/mira-assembler/wiki/Home/
-  license: ''
+  license: 'GNU General Public License v2 or later (GPLv2+)'
+  license_file: LICENCE
   summary: 'MIRA is a whole genome shotgun and EST sequence assembler for Sanger, 454, Solexa (Illumina), IonTorrent data and PacBio (the later at the moment only CCS and error-corrected CLR reads)'
 
 build:
-  number: 1
-  skip: True # [osx]
+  number: 2
+  skip: True # [not linux64 or osx]
```

Comment:
Well spotted - I was making assumptions about the operator precedence, and should probably have written ``# [not linux64 and not osx]`` here?

If currently BioConda only supports Mac OS X and 64 bit Linux removing the conditional would be fine for now.

Presumably we don't do 32 bit Linux, nor anything interesting like ARM or ...?